### PR TITLE
Attempt #2 at UHM-4354 (Pt 1, just Mexico) Move Location, Location Ta…

### PR DESCRIPTION
**Pull request by brandones**
…g, and Lcation Attribute Types installation out of PIH core and into config repos

Revert "Revert "UHM-4354 (Pt 1, just Mexico) Move Location, Location Tag, and Lcation Attribute Types installation out of PIH core and into config repos (#247)""

This reverts commit 3dce08d9575c8790d4807049a0336fd67be5871b.

This restores 360c72a3182b4f169f8c98dd96b9ef013a0346c6.

Fixes test failures introduced by that change.

### CODE REVIEW

File: api/src/test/java/org/openmrs/module/pihcore/PihCoreActivatorTest.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ public Properties getRuntimeProperties() {
     @Before
     public void setUp() throws Exception {
         initializeInMemoryDatabase();
-        executeDataSet("openmrsCoreDataset.xml");
+        executeDataSet("requiredDataTestDataset.xml");
```
Comment (brandones) [inline api/src/test/java/org/openmrs/module/pihcore/PihCoreActivatorTest.java:69]: This is new

File: api/src/test/java/org/openmrs/module/pihcore/liquibase/PrintingIDCardStatusChangeSetTest.java [hunk 2]
```diff
@@ -84,7 +84,7 @@ public void before() throws Exception {
 		database.setDatabaseChangeLogTableName("LIQUIBASECHANGELOG");
 		database.setDatabaseChangeLogLockTableName("LIQUIBASECHANGELOGLOCK");
 
-		executeDataSet("openmrsCoreDataset.xml");
+		executeDataSet("requiredDataTestDataset.xml");
```
Comment (brandones) [inline api/src/test/java/org/openmrs/module/pihcore/liquibase/PrintingIDCardStatusChangeSetTest.java:87]: This is new

File: api/src/test/resources/requiredDataTestDataset.xml [hunk 3]
```diff
@@ -46,8 +46,26 @@
                              name="Name to print on ID card" description="Name to print on ID card"
                              datatype="org.openmrs.customdatatype.datatype.FreeTextDatatype" retired="0" creator="1"
                              date_created="2006-01-18 00:00:00.0" uuid="a5fb5770-409a-11e2-a25f-0800200c9a66"/>
+    <location_attribute_type location_attribute_type_id="1" name="Default Label Printer"
```
Comment (brandones) [inline api/src/test/resources/requiredDataTestDataset.xml:49]: And these changes are new

**Comment by brandones** [comment]
Marked the new changes in comments.

File: api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/LocationBundle.java [hunk 4]
```diff
@@ -6,7 +6,6 @@
 import org.springframework.stereotype.Component;
 
 @Component
-@Requires({ LocationTagBundle.class, LocationAttributeTypeBundle.class} )
 public class LocationBundle extends AbstractMetadataBundle {
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/LocationBundle.java:9]: Does this class need to stay around at all?

Comment (mogoodrich) [reply api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/LocationBundle.java:9]: Oh, right, it does until we have every site moved over to Iniz-based loading

**Review by mogoodrich — APPROVED** [review]
LGTM, thanks @brandones !

**Pull request merged**